analyses/tcga-capture-kit-investigation/README.md

```diff
@@ -0,0 +1,50 @@
+# TCGA brain tumor WXS capture kit investigation
+This is an investigation of the TMB discrepancy between PBTA and TCGA data. 
+
+## Background
+To perform the analysis of [Tumor Mutation Burden Compare to TCGA](https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/tmb-compare-tcga), 160 tumor/normal paired TCGA WXS data had been selected and processed. The input BAM file manifest is on the `pbta-tcga-manifest.tsv` and is available in both V14 and V15 release. The MC3 WXS BED file [`gencode.v19.basic.exome.bed`](https://gdc.cancer.gov/about-data/publications/mc3-2017) was used as the capture kit region for both mutation calling and TMB calculation. It turns out the adult-based TCGA has overall lower TMB compared to pediatric-based PBTA data, and on top of that the Lancet somatic calling outputs much more low VAFs. To better figure out those issues, we started this investigation.
+
+We started by looking only the "codon impact mutation" where we only selected the `Mutation|Splice|Silent` mutations.
```

Comment:
agreed on this. also has been discussed here too: https://github.com/AlexsLemonade/OpenPBTA-analysis/pull/548#issuecomment-591564336

Reply:
ah right, so i just recalled, after the first quick try on those snp mutations and candace's comments on snp-vs-indel effects, i did a quick count locally for both too and did an "eye-ball" check and see the same trend, but never push to anywhere though. just put it on gist in case we'd like poke more https://gist.github.com/yuankunzhu/a7eb2a685113249ca6ac0fe47f9023a2